### ISSUE TITLE
spaces between module name and its content

### ISSUE THREAD

**Issue by aml987** [GavickPro/Magazine-Free-Joomla-Template#4]
hi, how can I remove this white spaces between module's title and its content?

---

![capture](https://cloud.githubusercontent.com/assets/7550942/2959850/c87d8298-dab6-11e3-8818-2181b0976a73.GIF)

**Comment by bkrztuk** [comment]
Could you give me your site URL ? It is always easier to check CSS/HTML code on live site.

**Comment by aml987** [comment]
www.sycourse.com/about
I Assigned it only for one page

**Comment by aml987** [comment]
I read this http://www.gavick.com/documentation/joomla-templates/customizing-joomla-template/how-to-change-paddings-margins-font-size-etc/
and it was very useful, thank you,,,

**Comment by aml987** [comment]
I edited css files on-line, but it doesn't apply directly, can I know why?

**Comment by bkrztuk** [comment]
I've visited your site and just can't see this part from → [url=http://cl.ly/image/1I2m322q1405]screenshot[/url] Could you you mark this area again?

**Comment by aml987** [comment]
the pic in first comment was in local server,
I'v read this article and it was useful,
http://www.gavick.com/documentation/joomla-templates/customizing-joomla-template/how-to-change-paddings-margins-font-size-etc/
thank you

**Comment by bkrztuk** [comment]
Knowledge how to use firebug is very helpful when you want to customize template :)

**Comment by aml987** [comment]
thanks, it's very easy :+1:  I want to ask you , what the plug-in you used on chrome to display the script that used in the sites?
![capture](https://cloud.githubusercontent.com/assets/7550942/3005783/edc1e618-ddf2-11e3-86d2-b63a7f3b5a30.GIF)

**Comment by bkrztuk** [comment]
It is called Chrome Sniffer :) http://techdo.me/chrome-sniffer-a-must-have-chrome-extension-for-web-developers/

**Comment by aml987** [comment]
thank you again :+1:  , I want to send you the Arabic file language and the template after I edited it for Arabic users in your site,,, if you want that ...

**Comment by bkrztuk** [comment]
Sure we will include it in our repository.

**Comment by aml987** [comment]
how can i send you the file?

**Comment by bkrztuk** [comment]
If you can make it via GitHub - you need to fork repository you can just share the file via mail : bkrztuk[at]gavick.com

**Comment by aml987** [comment]
OK, I'm a new user in GitHub but I will try to add it here, and I will send you a copy of it, 
I want tp ask you one more question please: I tried to apply facebook login but I didn't success , I added API key and do all the steps , I turn it now in my website www.sycourse.com ,  can check it please?

**Comment by bkrztuk** [comment]
Your website shows some JS errors in console but it looks like it is not code which by default is inclued in template, did you add some scripts or modify gk.script.js file?

**Comment by aml987** [comment]
no , i didn't add any js files

**Comment by aml987** [comment]
and didn't  modify gk.script.js file

**Comment by bkrztuk** [comment]
Right now I can't see login with FB button, it is disabled ? » http://cl.ly/image/0R0U3q3r0G43

**Comment by aml987** [comment]
yes I disabled it yesterday because you didn't reply me, what about js errors? how can I fix it?
I send you a copy of template, did you open my email?

**Comment by bkrztuk** [comment]
Now I don't see this messages from 3rd party extensions, please just enable button and let me know. You language file is available here  » https://github.com/GavickPro/Magazine-Free-Joomla-Template/commit/271269cc59861a13bca0565c05945ff3df79ae9a

**Comment by aml987** [comment]
I enabled facebook login button , www.sycourse.com

**Comment by bkrztuk** [comment]
And the error is still visible, please look » http://cl.ly/image/0p0R2p1q0p0A

This probably is blocking other scripts and nothing is change when I click Login with facebook.

**Comment by aml987** [comment]
so, tell me what should I do please?

**Comment by bkrztuk** [comment]
One of your extensions add JavaScript code which product validation error, I'm unable to detect which one because the code is added inline. Did you modify the template head section by adding custom JavaScript code there?

**Comment by aml987** [comment]
I added support code to index.php file

**Comment by bkrztuk** [comment]
Some extension produce JS errors which block other scripts - in clean template there is no this kind of issue so probably it is problem related to your 3rd party extensions. The only way to find out what may be wrong is to disable them one by one and just check when console will be clean.
